### ISSUE TITLE
The `opengee-fusion` RPM is Missing a Dependency on `which`

### ISSUE THREAD

**Issue by tst-ppenev** [google/earthenterprise#938]
## Steps to Reproduce

1. Build Open GEE RPMs.
1. Have a Cent OS 7 instance that doesn't have the `which` command installed, such as Docker's `centos:7` image.
1. Install the Open GEE RPMs.
1. Run `/etc/init.d/gefusion status`.

## Observed Behavior

```
/etc/init.d/gefusion: line 78: which: command not found
gesystemmanager:  Not running.
/etc/init.d/gefusion: line 78: which: command not found
geresourceprovider:  Not running.
```

## Expected Behavior

```
gesystemmanager:  Running (PID 504).
geresourceprovider:  Running (PID 508).
```

## Notes

This issue can be resolved by adding a dependency on the `which` command, or by refactoring the `init.d` script to use the `type` shell build-in.  It should make little difference, but get some agreement before implementing.

**Comment by tst-ccamp** [comment]
I vote for switching to the `type` command. The fewer the dependencies, the better.

**Comment by tst-ccamp** [comment]
This was previously fixed in #1019, so closing
